feeds.xsh

```diff
@@ -0,0 +1,24 @@
+import asyncio
+import threading
+import time
+from feedy import Feedy
+
+app = Feedy('feeds.shelve')
+
+@app.add('https://stackoverflow.com/feeds/tag?tagnames=xonsh&sort=newest')
+def stackoverflow_question(info, body):
+    # site_title, site_subtitle, site_url, fetched_at, article_title, article_url, published_at, updated_at
+    msg = "New question on StackOverflow: {article_title} {article_url}".format(**info)
+    gitter_send_msg(msg)
+    send_irc_msg(msg)
+
+
+def run():
+    loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(loop)
+    while True:
+        app.run()
+        time.sleep(60*60*60)  # 1 hour
```

Comment:
It's generally polite to not poll RSS feeds too often. On the order of hours is conventional. This is, of course, 100% a matter of policy.

Although I think that's actually 60hrs. It should be `1*60*60`.